messaging/src/test/java/org/apache/falcon/messaging/JMSMessageConsumerTest.java

```diff
@@ -309,7 +298,6 @@ public void testJMSMessagesFromOozieForNonFalconWF() throws Exception {
         sendMessages(TOPIC_NAME, WorkflowExecutionContext.Type.WORKFLOW_JOB, false /* isFalconWF */);
 
         final BrokerView adminView = broker.getAdminView();
-
         Assert.assertEquals(adminView.getTotalDequeueCount(), 0);
```

Comment:
May be remove it here too to be consistent and also so the test doesn't fail later.